Source/WebCore/html/HTMLPreElement.cpp

```diff
@@ -56,7 +56,7 @@ void HTMLPreElement::collectPresentationalHintsForAttribute(const QualifiedName&
 {
     if (name == wrapAttr) {
         style.setProperty(CSSPropertyWhiteSpaceCollapse, CSSValuePreserve);
-        style.setProperty(CSSPropertyTextWrap, CSSValueWrap);
+        style.setProperty(CSSPropertyTextWrapMode, CSSValueWrap);
```

Comment:
Just confirming that we do not need to also set `text-wrap-style` here?

Reply:
Same here